Source/SteveSloth/Public/MyPermanentHealthPickup.h

```diff
@@ -0,0 +1,39 @@
+/****************************************************************************************
+ * Copyright:
+ * Name: Guy Lapensee
+ * Script: AMyPermanentHealthPickup
+ * Date: May 16, 2024
+ * Description:Leaf  pickup for 25 permanent health boost
+ * TODO: Permanently add health to the players health bar 
+ * Known Bugs:
+ ****************************************************************************************/
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "ItemBaseClass.h"
+#include "MyPlayer.h"
+#include "MyPermanentHealthPickup.generated.h"
+
+/**
+ * 
+ */
+UCLASS()
+class STEVESLOTH_API AMyPermanentHealthPickup : public AItemBaseClass
+{
+	GENERATED_BODY()
+	
+public:
+	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Item Attributes")
+	float HealthIncrease;
+
+private: // PRIVATE VARIABLES
+	AItemBaseClass* ItemClass;
+	AMyPlayer* PlayerClass;
```

Comment:
You dont need either of these as you are inheriting for BaseItemClass and the player is gotten a different way.